src/CodeCoverage/Exception/InvalidArgumentException.php

```diff
@@ -21,7 +21,7 @@ class PHP_CodeCoverage_InvalidArgumentException extends InvalidArgumentException
      */
     public static function create($argument, $type, $value = null)
     {
-        $stack = debug_backtrace(false);
+        $stack = debug_backtrace(PHP_VERSION_ID < 50306 ? false : 0);
```

Comment:
Thanks for the note on 5.6.

The parameter is a bitmask, so `! DEBUG_BACKTRACE_PROVIDE_OBJECT` wouldn't have the desired effect.